pom.xml

```diff
@@ -33,40 +33,74 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-security</artifactId>
+			<version>3.3.4</version>
```

Comment:
Is there a specific reason why all the spring-boot-starter versions are now explicitly mentioned? They should already be automatically configured by the ` spring-boot-starter-parent`.